pkgs/by-name/de/delineate/package.nix

```diff
@@ -0,0 +1,95 @@
+{
+  appstream,
+  buildNpmPackage,
+  cargo,
+  cmake,
+  desktop-file-utils,
+  fetchFromGitHub,
+  gtk4,
+  gtksourceview5,
+  lib,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  rustPlatform,
+  rustc,
+  stdenv,
+  webkitgtk_6_0,
+  wrapGAppsHook4,
+}:
+let
+  d3-graphviz = buildNpmPackage rec {
+    pname = "d3-graphviz";
+    version = "5.6.0";
+
+    src = fetchFromGitHub {
+      owner = "magjac";
+      repo = "d3-graphviz";
+      tag = "v${version}";
+      hash = "sha256-MZhAzR6+GIBTsLPJq5NqaEPHjiBMgYBJ0hFbDPNPgFk=";
+    };
+
+    npmDepsHash = "sha256-J1kptumP/8UoiYDM+AJOYUne0OSkMXCTAXW3ZmavU4E=";
+
+    # keep the devDependencies, as Delineate imports d3 via node_modules
+    # https://github.com/SeaDve/Delineate/blob/v0.1.0/data/graph_view/index.html#L10-L11
+    npmPruneFlags = "--include=dev";
+
+    meta = {
+      description = "Graphviz DOT rendering and animated transitions using D3";
+      homepage = "https://github.com/magjac/d3-graphviz";
+      license = lib.licenses.bsd3;
+    };
```

Comment:
I'm not sure the meta section is really necessary if this is just a helper package